app/addons/activetasks/routes.js

```diff
@@ -15,29 +15,24 @@ import FauxtonAPI from "../../core/api";
 import ActiveTasksResources from "./resources";
 import ActiveTasksComponents from "./components.react";
 import Actions from "./actions";
+import Layout from './layout';
 
 var ActiveTasksRouteObject = FauxtonAPI.RouteObject.extend({
   selectedHeader: 'Active Tasks',
-  layout: 'one_pane',
+  layout: 'empty',
+  hideBreadCrumbs: true,
```

Comment:
the way the new crumbs work now is a bit different, they just don't show up if no crumbs are provided. i don't like the implicity, but it helps to declutter route objects from functionality in order to get rid of them soon.

I see that the new <CrumbsWrapper /> supports the same functionality, too:

```js
const CrumbsWrapper = ({crumbs}) => {
   if (!crumbs || crumbs.length === 0) {
     return null;
   }

```

so i'm not sure if this is needed here and part of a new migration strategy?